x/ibc/07-tendermint/types/codec.go

```diff
@@ -26,6 +26,10 @@ func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 		(*clientexported.ClientState)(nil),
 		&ClientState{},
 	)
+	registry.RegisterImplementations(
```

Comment:
we also need to register the interface on 02-client:

```golang
registry.RegisterInterface(
	"cosmos_sdk.ibc.v1.client.ConsensusState",
	(*exported.ConsensusState)(nil),
)
```

Reply:
will do this in the followup pr migrating 02-client :+1: